types/chai-as-promised/index.d.ts

```diff
@@ -188,16 +188,16 @@ declare namespace Chai {
     }
 
     interface PromisedProperty {
-        (name: string, value?: any, message?: string): PromisedAssertion;
+        (name: string | Symbol, value?: any, message?: string): PromisedAssertion;
```

Comment:
May want to consider using the primitive `symbol` rather than `Symbol` in each of these changes.

Reply:
Sure! Happy to use whatever's preferred.